quiz.js

```diff
@@ -1 +1,83 @@
-//Your quiz functionality goes here
\ No newline at end of file
+//Your quiz functionality goes here
+let rightChoices = 0;
+let quesoNum = 0;
+
+function startGame()
+{
+    quesoNum = 0;
+    //displayQuestions();
+    let quizQuestions = getQuestions();
+    console.log(quizQuestions[0]);
```

Comment:
you should remove this line, it causes an error if nothing is in session storage